web/src/ApplicationEditPage.js

```diff
@@ -145,7 +145,7 @@ class ApplicationEditPage extends React.Component {
           application: application,
         });
 
-        this.getCerts(application.organization);
+        this.getCerts(application.isShared ? this.props.account.owner : application.organization);
```

Comment:
Is this the only file to change? other objects?

This mess should be handled by XXXBackend.js, so messed up here in UI code